app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -104,7 +104,11 @@
     <string name="template_empty">Projeto vazio</string>
     <string name="template_navigation_drawer">Projeto navigation drawer</string>
     <string name="template_libgdx">Jogo LibGDX</string>
-    <string name="template_description_libgdx">Projeto de jogo LibGDX configurado para renderizar uma imagem como exemplo.</string>
+    <string name="template_description_libgdx">Projeto de jogo LibGDX configurado para renderizar uma imagem como exemplo.</string>
+    <string name="template_login_activity">Projeto Tela de login</string>
+    <string name="template_login_description">Um projeto básico que contem uma tela de Login pré-construido</string>
```

Comment:
Remove strings _"template_login_activity"_ and _"template_login_description"_ from this list.